lib/datasource/remote/polkadot_api/polkadot_repository.dart

```diff
@@ -167,22 +175,24 @@ class PolkadotRepository extends KeyRepository {
       ///
       await startService().timeout(const Duration(seconds: 20));
 
-      print("DONE SERVICE");
+      print("DONE");
 
       if (state.isConnected) {
         eventBus.fire(const ShowSnackBar("Network reconnected."));
       }
     } catch (error) {
       print("reconnect error $error");
       rethrow;
+    } finally {
+      startKeepAliveTimer();
     }
   }
 
   Future<Result<Account?>> getIdentity(String address) async {
     try {
       print("get identity for $address");
```

Comment:
i removed isReady since it has an unclear meaning that was sometimes used in the wrong way, causing bugs